### PR TITLE
Added event user class for handling relationships between them

### PR DESCRIPTION
**Pull request by RBarnabass**

### CODE REVIEW

File: src/main/java/com/softserve/actent/model/entity/EventUser.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+    @ManyToOne
+    private Event event;
+
+    @ManyToOne
+    private User user;
+
+    @Enumerated(EnumType.STRING)
+    private EventUserType type;
+}
```
Comment (thebadpanda) [inline src/main/java/com/softserve/actent/model/entity/EventUser.java:32]: add newline at the end of file

File: src/main/java/com/softserve/actent/model/entity/EventUser.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+@Entity
+@Table(name = "events_users")
+public class EventUser {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @ManyToOne
+    private Event event;
```
Comment (thebadpanda) [inline src/main/java/com/softserve/actent/model/entity/EventUser.java:25]: the default fetch type of @ManyToOne is EAGER. Do you need it here? If so, than explain why.

File: src/main/java/com/softserve/actent/model/entity/EventUser.java [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+    @ManyToOne
```
Comment (thebadpanda) [inline src/main/java/com/softserve/actent/model/entity/EventUser.java:27]: the default fetch type of @ManyToOne is EAGER. Do you need it here? If so, than explain why.

**Review by md1guy — APPROVED** [review]

File: src/main/java/com/softserve/actent/model/entity/EventUser.java (hunk 1 above)
Comment (RBarnabass) [reply src/main/java/com/softserve/actent/model/entity/EventUser.java:32]: w?

File: src/main/java/com/softserve/actent/model/entity/EventUser.java (hunk 3 above)
Comment (RBarnabass) [reply src/main/java/com/softserve/actent/model/entity/EventUser.java:27]: It's temporary, after detail tests I can configure it!

File: src/main/java/com/softserve/actent/model/entity/EventUser.java (hunk 2 above)
Comment (RBarnabass) [reply src/main/java/com/softserve/actent/model/entity/EventUser.java:25]: It's temporary, after detail tests I can configure it!

**Review by octap16 — APPROVED** [review]

**Review by Senchakmen — APPROVED** [review]

**Pull request merged**